### ISSUE TITLE
Add OpenSRP Service Module

### ISSUE THREAD

**Issue by cKellyDesign** [onaio/reveal-frontend#208]
This service will allow us to connect directly to the OpenSRP server to GET, PUSH, and PUT entities, starting with Plans.

- [x] GET Plans
- [x] GET Plan by Unique ID
- [x] PUSH New Plan
- [x] PUT Updated Plan

Unblocks: #141, #142
